Scripts/Game/ODIN/Helpers/ODIN_TypeHelper.c

```diff
@@ -0,0 +1,26 @@
+/**
+Helper class with static functions for helping type match
+*/
+class ODIN_TypeHelper 
+{
+	static string VehicleTypeName = "Vehicle";
+	static string CharacterTypeName = "SCR_ChimeraCharacter";
+	
+	static bool IsUnit(IEntity entity)
+	{
+		string type = entity.ClassName();
+		if (type.Contains(ODIN_TypeHelper.CharacterTypeName))
```

Comment:
You should check the type by casting the instance. If it can't be casted, `null` will be returned.

Reply:
To ensure you don't pass a null as entity? 
Would you then suggest the function returns false, or return null? Technically null is not a unit, but doesn't really give feedback to the caller that the call was invalid as entity is null.

Reply:
I would even go as far as say that that `ODIN_TypeHelper` is completely unnecessary. You can replace the methods essentially by `if (Vehicle.Cast(entity))` and so on.

Reply:
Handles `null` properly and is way more efficient

Reply:
I did not know you could just cast it into ChimeraCharacter and Vehicle to test it. Will change to use that instead. Only made TypeHelper because I couldn't find an easy built-in way to check it in a 1-liner.

Reply:
No worries, now you know 😄

Reply:
Changed to the cast calls and removed typehelper. Tested it and seems to work no problem.